components/Navigation.css

```diff
@@ -0,0 +1,54 @@
+/* color variables */
+
+:root {
```

Comment:
You can separate the CSS variable in different file

Reply:
done